Dockerfile.build

```diff
@@ -1,4 +1,6 @@
-FROM golang:1.4.3-cross
+FROM golang:1.5
+
+ENV USER root
 
 # Install pkcs11 deps
```

Comment:
These probably aren't necessary as `script/build` builds with `-tags nopkcs11` (because there's not support across all platforms?)